### PR TITLE
Contact Us section

### PR DESCRIPTION
**Pull request by PranavBarthwal**
Built the contact us section.
Screenshots:
Screenshot 2024-01-16 115203
WhatsApp Image 2024-01-15 at 23 03 42_8ce960cc

### CODE REVIEW

File: index.html
Comment (ritika728) [inline index.html:1]: Submit button is working even when all the fields are empty. Can you make all the fields of the form as required ?

Comment (ritika728) [inline index.html:1]: Also, the alignment of the navbar is disturbed. Please look into that

**Comment by PranavBarthwal** [comment]
@ritika728  I have made the all the required changes. kindly review my PR. Thanks.

Screenshot 2024-01-16 145718

**Comment by ritika728** [comment]
@PranavBarthwal Looks good to merge
Thank you for your contribution!

**Comment by ritika728** [comment]
Done

**Pull request merged**